src/app/containers/Toggle/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { string, element } from 'prop-types';
+import useToggle from './useToggle';
+
+const ToggleContainer = ({ toggleName, FallbackComponent, children }) => {
+  const { enabled } = useToggle(toggleName);
+
+  if (enabled) {
+    return children;
+  }
+
+  if (FallbackComponent) {
```

Comment:
technically you could just return `FallbackComponent` without the `if` here, as it default to null, but feel free to leave it

Reply:
I think wrapping it in the if statement is a bit more descriptive on what is actually happening.